### PR TITLE
Don't check out the projects

### PR DESCRIPTION
**Pull request by milt**
Let them do that on their own, the two should not couple

### CODE REVIEW

File: setup-env/action.yml [hunk 1]
```diff
@@ -26,8 +26,6 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - uses: actions/checkout@v2
-
```
Comment (kelvinqian00) [inline setup-env/action.yml:30]: What exactly was the issue here? Was this action clashing with something?

Comment (milt) [reply setup-env/action.yml:30]: Mainly it is to give the user more control. Perhaps they want to do something post-checkout but pre-deps. The point is the two shouldn't be coupled

Comment (milt) [reply setup-env/action.yml:30]: You'll see similar commits across lots of github actions after people use them for a bit and figure this out.

**Review by kelvinqian00 — CHANGES_REQUESTED** [review]
Don't forget to update the README (currently it says that checking the project out is one of the steps).

File: setup-env/action.yml (hunk 1 above)
Comment (milt) [reply setup-env/action.yml:30]: https://github.com/yetanalytics/actions/pull/1/commits/d64f58204305b3b1aab18deb2a3a8a3fd9f80ceb

**Review by kelvinqian00 — APPROVED** [review]

**Pull request merged**